scripts/concent_acceptance_tests/base.py

```diff
@@ -381,3 +380,8 @@ def requestor_put_deposit(self, price: int):
     def provider_put_deposit(self, price: int):
         amount, _ = helpers.provider_deposit_amount(price)
         return self.put_deposit(self.provider_sci, amount)
+
+    @staticmethod
+    def _blockchain_sleep(sleep_time=60):
```

Comment:
```suggestion
    def blockchain_sleep(sleep_time=60):
```

This method is not internal to base class